lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -251,6 +276,15 @@ def test_enabled_block_types_with_assignments(self):
                     self.assertGreater(now, assignment.date)
                     for html_tag in assignment_title_html:
                         self.assertIn(html_tag, assignment_title)
+                elif 'No start date' == assignment_title:
```

Comment:
Are these supposed to be internationalized or is this fine because it's just test content and we key i18n off of  english strings

Reply:
I don't believe we internationalize our test files